test/integration/admin/sidebar/clinician-sidebar.js

```diff
@@ -568,14 +566,14 @@ context('clinician sidebar', function() {
       .find('[data-name-region]')
       .should('contain', 'name error')
       .find('.js-input')
-      .should('have.css', 'border-color', stateColors.error);
```

Comment:
what is the issue here?

Reply:
The CSS property was blank in Firefox. I realized that we have the `.has-error` class and could just test for that. But I can troubleshoot the blank CSS property if we want to keep that test in there.

Reply:
we do this same test in other places.. I think it was failing there too though.. but yeah it'd be nice to determine why `has.css` isn't working

Reply:
I think instead of needing this comment everywhere let's add this note to the integration readme under state colors and then just fallback on tests failing